test/integration/guest_can_view_items_sorted_by_artist_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.